src/main/resources/application.properties

```diff
@@ -28,6 +30,7 @@ clientSecret=
 #HIP
 getPatientPath=http://localhost:8081/v1/patients
 
+logging.level.com.nha.abdm.wrapper=DEBUG
```

Comment:
By default, we can have the `INFO` logs enabled, developers can enable `DEBUG` mode for debugging, same for `logging.level.org.springframework.web=DEBUG` as well.

Reply:
Its good to have debug enabled in initial stages. So that we can co-ordinate with integrators when issues found. We can turn it to INFO once we have stable release.

Reply:
Ok Agreed. Can we disable for springframwork, I am assuming it might be very verbose. If it is doing minimal logging then we can retain this.

Reply:
Yes, it is not verbose since its for web.